Configuring multiple helm chart repositories

Preeticp opened the pull request:
This PR covers the dev issue: https://issues.redhat.com/browse/APPSVC-385 and docs issue: https://issues.redhat.com/browse/RHDEVDOCS-2175
This is relevant to 4.6.

Preeticp commented:
@pedjak @gajanan-more request you to review this PR, please.

pedjak commented on modules/helm-creating-role-bindings-to-access-multiple-chart-repositories.adoc:
```diff
@@ -0,0 +1,115 @@
+
+.Procedure
+
+* Provide regular users access to browse Helm chart repositories by creating the following role and role binding:
```
this is already the default state in the cluster, hence there is no need to document it.

pedjak commented on modules/helm-creating-role-bindings-to-access-multiple-chart-repositories.adoc:
```diff
@@ -0,0 +1,115 @@
+  name: azure-helm-repo-viewer
+----
+
+* For chart repositories where certificates are needed, provide users with read access to the `helm-ca-cert` configmap and `helm-tls-configs` secret in the `openshift-config` namespace:
```
this needs to stay.

pedjak reviewed (commented):
by default all authenticated users have access to all configured charts.

Preeticp commented:
@pedjak I have removed the redundant information and moved the required role binding part to the Creating credentials and CA certificates to add Helm chart repositories section as a last step. Please take a look: https://helm-multi-rep--ocpdocs.netlify.app/openshift-enterprise/latest/cli_reference/helm_cli/configuring-custom-helm-chart-repositories.html

Preeticp commented:
@gajanan-more, PTAL, I have fixed the mention of Developer Perspective too.

gajanan-more commented:
/lgtm

akashshinde commented:
/lgtm

bmcelvee commented on cli_reference/helm_cli/configuring-custom-helm-chart-repositories.adoc:
```diff
@@ -0,0 +1,14 @@
+
+toc::[]
+
+The *Developer Catalog*, in the *Developer* perspective of the web console, displays the Helm charts available in the cluster. By default, it lists the Helm charts from the link:https://redhat-developer.github.io/redhat-helm-charts/index.yaml[Red Hat Helm chart repository]. As a cluster administrator, you can add multiple Helm chart repositories, apart from the default one, and display the Helm c ...
```
Is the link to the Red Hat Helm chart repository intended to open a file or direct to the actual repo?

Preeticp replied:
Modified this a bit.

bmcelvee commented on modules/helm-adding-helm-chart-repositories.adoc:
```diff
@@ -0,0 +1,43 @@
+
+[id="adding-helm-chart-repositories_{context}"]
+= Adding custom Helm chart repositories
+
```
Add short description of procedure (per the template)

bmcelvee commented on modules/helm-creating-credentials-and-certificates-to-add-helm-repositories.adoc:
```diff
@@ -0,0 +1,87 @@
+.Procedure
+To configure the credentials and certificates, and then add a Helm chart repository using the CLI:
+
+. In the `openshift-config` namespace, create a `configmap` with a custom CA certificate in PEM encoded format, and store it under the `ca-bundle.crt` key within the configmap.
```
s/configmap./configmap:

bmcelvee commented on modules/helm-creating-credentials-and-certificates-to-add-helm-repositories.adoc:
```diff
@@ -0,0 +1,87 @@
+-n openshift-config
+----
++
+. In the `openshift-config` namespace, create a `secret`  to add the Client TLS Configurations. The client certificate and key must be in PEM encoded format and stored under the keys `tls.crt` and `tls.key`, respectively.
```
Take this one or leave it :)

I see why you structured this one with the detail sentence following the action step (hence the take it or leave it) but for consistency, consider putting the command immediately following the action. E.g., 

. In the `openshift-config` namespace, create a `secret`  to add the Client TLS Configurations:
<command>
The client certificate and key must be in PEM encoded format and stored under the keys `tls.crt` and `tls.key`, respectively.

bmcelvee commented on modules/helm-creating-credentials-and-certificates-to-add-helm-repositories.adoc:
```diff
@@ -0,0 +1,87 @@
++
+[source,terminal]
+----
+oc create secret generic helm-tls-configs \
```
s/oc/$ oc

bmcelvee commented:
Just a few questions/suggestions, otherwise LGTM! And just a reminder to squash commits before merging. :)

Preeticp commented:
Thank you @bmcelvee I have fixed all the comments, PTAL

bmcelvee commented:
LGTM - thank you!!

bmcelvee commented:
/cherrypick enterprise-4.6
